pkg/client/config.go

```diff
@@ -76,6 +82,33 @@ func SaveConfig(config map[string]string) error {
 	return json.NewEncoder(configFile).Encode(&config)
 }
 
+func (c VeleroConfig) Namespace() string {
+	val, ok := c[ConfigKeyNamespace]
+	if !ok {
+		return ""
+	}
+	ns, ok := val.(string)
+
```

Comment:
nit: rm blank line (L91)

Reply:
IMO not having that line makes it crowded, but I can remove it.